src/uu/date/src/date.rs

```diff
@@ -330,6 +408,13 @@ pub fn uu_app() -> Command {
                 .value_hint(clap::ValueHint::FilePath)
                 .help("like --date; once for each line of DATEFILE"),
         )
+        .arg(
+            Arg::new(OPT_RESOLUTION)
+                .long(OPT_RESOLUTION)
+                .value_name("RESOLUTION")
+                .help("output the available resolution of timestamps")
+                .action(ArgAction::SetTrue),
```

Comment:
> When you implement it, please make sure that you set overrides_with, so that date --resolution --resolution also works :)

Please implement and also test this.